testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1461,8 +1461,9 @@
   new_hostname = old_hostname + '2'
   log "New short hostname: #{new_hostname}"
   server_node.run("sed -i 's/#{old_hostname}/#{new_hostname}/g' /etc/hostname &&
-  echo '#{server_node.public_ip} #{server_node.full_hostname} #{old_hostname}' >> /etc/hosts &&
-  echo '#{server_node.public_ip} #{new_hostname}#{server_node.full_hostname.delete_prefix(server_node.hostname)} #{new_hostname}' >> /etc/hosts")
+                   hostname #{new_hostname} &&
+                   echo '#{server_node.public_ip} #{server_node.full_hostname} #{old_hostname}' >> /etc/hosts &&
+                   echo '#{server_node.public_ip} #{new_hostname}#{server_node.full_hostname.delete_prefix(server_node.hostname)} #{new_hostname}' >> /etc/hosts")
```

Comment:
Can we get some debugging output here, please? I would really like to have some feedback on stdout in the console output top better and quicker understand when something fails in the future.

Reply:
We have, on previous lines. Here is the output:
```
  Scenario: Change hostname back and reboot server            # features/secondary/srv_rename_hostname.feature:41
      This scenario ran at: 2023-08-29 15:42:56 +0200
Initializing a twopence node for 'server'.
Host 'server' is alive with determined hostname suma-head-srv and FQDN suma-head-srv.mgr.suse.de
Node: suma-head-srv, OS Version: 15-SP4, Family: sles
    When I change back the server's hostname                  # features/step_definitions/command_steps.rb:1496
Initializing a twopence node for 'server'.
Host 'server' is alive with determined hostname suma-head-srv and FQDN suma-head-srv.mgr.suse.de
Node: suma-head-srv, OS Version: 15-SP4, Family: sles
machine: 10.137.210.177 went down
machine: 10.137.210.177 network is up
machine: 10.137.210.177 ssh is up
    And I reboot the server through SSH                       # features/step_definitions/command_steps.rb:1413
      Server spacewalk service is up
Initializing a twopence node for 'server'.
Host 'server' is alive with determined hostname suma-head-srv and FQDN suma-head-srv.mgr.suse.de
Node: suma-head-srv, OS Version: 15-SP4, Family: sles
    And I run spacewalk-hostname-rename command on the server # features/step_definitions/command_steps.rb:1471
      [
        "26"
      ]Validating IP ... OK
      =============================================
      hostname: suma-head-srv
      old hostname: suma-head-srv.mgr.suse.de
      ip: 10.137.210.177
      =============================================
      =============================================
      hostname: suma-head-srv.mgr.suse.de
      =============================================
      Currently Report DB Host is: suma-head-srv.mgr.suse.de
      suma-head-srv.mgr.suse.de is already correct. Nothing to do.
      Do you want to change it to suma-head-srv.mgr.suse.de ? [y/N]: 
      Server: spacewalk service is up
      This scenario took: 321 seconds
```
If that's not enough, please tell me what I should add.

Reply:
We see the new hostname which is enough. Thank you for the clarification.

Reply:
Done.